### PR TITLE
definition of a representable covariant family

### PR DESCRIPTION
**Pull request by jonalfcam**
I'm starting work on 9.10 and probably the definition of Yoneda embedding (def 9.3).

### CODE REVIEW

**Comment by fizruk** [comment]
Regarding the Yoneda embedding, if you haven't started, it seems #100 has it.

**Comment by jonalfcam** [comment]
Great!  I hadn't seen anyone claim it on the issues page. I won't have to do it!

**Comment by cesarbm03** [comment]
Hey Jonathan, I just saw this now. I made some progress on the Yoneda embedding over the weekend. I should comment on the issue then. Sorry about it.

File: src/simplicial-hott/09-yoneda.rzk.md [hunk 1]
```diff
@@ -1155,3 +1155,43 @@ proven, just with an equivalent type in the domain of the evaluation map.
       ( contra-dependent-yoneda-lemma'
           A is-segal-A a C is-contravariant-C)
 ```
+
+## Representable Families
+
+A covariant family is representable if it is fiberweise equivalent to covariant
+homs. In order to check if this is the case, it is not necessary to know if the
+family if covariant or not.
```
Comment (emilyriehl) [inline src/simplicial-hott/09-yoneda.rzk.md:1163]: typo "if" -> "is"

File: src/simplicial-hott/09-yoneda.rzk.md [hunk 2]
```diff
@@ -1155,3 +1155,43 @@ proven, just with an equivalent type in the domain of the evaluation map.
...
+#def equiv-for-is-representable-family
+  ( A : U)
+  ( C : A → U)
+  ( is-rep-C : is-representable-family A C)
+  : (x : A) → (hom A (first is-rep-C) x) → (C x)
+  := \ x → first((second (is-rep-C)) x)
+```
+
+RS Proposition 9.10 gives an if and only if condition for a covariant family
+$C : A → \mathcal{U}$ to be representable. The condition is that the type
```
Comment (emilyriehl) [inline src/simplicial-hott/09-yoneda.rzk.md:1186]: I think `#!rzk C : A  → U` will look better than the LaTeX here.

File: src/simplicial-hott/09-yoneda.rzk.md [hunk 3]
```diff
@@ -1155,3 +1155,43 @@ proven, just with an equivalent type in the domain of the evaluation map.
...
+$\sum_{x : A} C(x)$ has an initial object. For convenience, we give this
```
Comment (emilyriehl) [inline src/simplicial-hott/09-yoneda.rzk.md:1187]: Ditto use a rzk code block instead of LaTeX.

**Review by emilyriehl — CHANGES_REQUESTED** [review]

**Comment by emilyriehl** [comment]
Minor comments about the markdown portion. Otherwise I think this is okay to merge.

**Comment by jonalfcam** [comment]
Thanks for taking a look. I'll try to default to rzk code blocks in the future.

**Review by emilyriehl — APPROVED** [review]

**Comment by jonalfcam** [comment]
Looking at what it takes to formalize this, it seems there is a need for a bunch of "hom behaves well with respect to equivalences of underlying types" statements. That is, if $A \simeq B$ then we would like $\text{hom}_A (a, a') \simeq \text{hom}_B (b, b')$. This statement then requires some statements about "natural equivalences." I believe to prove this one needs some trivial but useful statements about the functoriality of hom with respect to equality. For instance, under no assumptions about $A$, $\text{hom}_A (a, x) = \text{hom}_A (a, y)$ if $x = y$. Given their somewhat basic nature, I'd be tempted to include these shortly after the definition of hom. Then it is reasonably easy to prove that every property of hom is respected by equivalence of types. 

It is also possible that I'm being dense and I've missed that this is already implemented or doesn't need to be.

**Comment by jonalfcam** [comment]
It is also possible that they are trivial enough to not require their own names, but it seems somewhat useful.

**Comment by jonalfcam** [comment]
Eh. I was trying to do something too general, and I think this needs univalence. I will revert to working with less general types.

**Pull request merged**